managed/CounterStrikeSharp.API/Core/API.cs

```diff
@@ -337,6 +337,16 @@ public static int GetMaxClients(){
 			}
 		}
 
+        public static float GetGameframeTime(){
```

Comment:
might be better as `GetGameFrameTime` (use `GET_GAME_FRAME_TIME` in the yaml)

Reply:
Yeah I agree, I just used the name that was already in the natives cpp.